recipe/meta.yaml

```diff
@@ -28,7 +29,6 @@ requirements:
     - pip
   run:
     - python
```

Comment:
```suggestion
    - python
    - numpy <2
```

Reply:
Hi !
What's the point of pinning "numpy<2" on a pull request whose purpose is to build for numpy2.0 ?
Did it break any tests?

Reply:
I missed this comment, but I guess I was trying to follow these instructions:
> You should check that the upstream package explicitly supports numpy 2.0, otherwise you
need to add a - numpy <2 run requirement until that happens